filepathfilter/filepathfilter.go

```diff
@@ -61,30 +63,20 @@ func (f *Filter) Allows(filename string) bool {
 		return true
 	}
 
-	if len(f.include)+len(f.exclude) == 0 {
-		return true
+	var matched bool
+	for _, inc := range f.include {
+		if matched = inc.Match(filename); matched {
+			break
+		}
 	}
 
-	cleanedName := filepath.Clean(filename)
-
-	if len(f.include) > 0 {
-		matched := false
-		for _, inc := range f.include {
-			matched = inc.Match(cleanedName)
-			if matched {
-				break
-			}
-		}
-		if !matched {
-			return false
-		}
+	if !matched && len(f.include) > 0 {
+		return false
```

Comment:
Great suggestion, I definitely prefer `included` to `matched`, since `included` communicates inclusion, but `matched` only communicates ambiguity. "Matched what?"

I adopted this suggestion by renaming this to `included` in c48f8178.